index.html

```diff
@@ -0,0 +1,115 @@
+
+    <main>
+      <section class="main-banner">
+        <banner>
+          <div class="main-banner__content">
+            <h1>
+              일상의 모든 물건을<br />
+              거래해 보세요
+            </h1>
+            <a href="/items"><button id="search-btn">구경하러 가기</button></a>
+          </div>
+          <img src="img/Img_home_top.png" />
+        </banner>
+      </section>
+
+      <section class="main-articles">
+        <article class="main-article">
+          <div class="main-article-container">
+            <img src="img/Img_home_01.png" />
+            <div class="main-article__text">
+              <h5>Hot item</h5>
+              <h2>
+                인기 상품을<br />
+                확인해 보세요
+              </h2>
+              <p>
+                가장 HOT한 중고거래 물품을<br />
+                판다 마켓에서 확인해 보세요.
+              </p>
+            </div>
+          </div>
+        </article>
+        <article class="main-article">
+          <div class="main-article-container">
+            <div class="main-article__text main-article__text2">
+              <h5>Search</h5>
+              <h2>
+                구매를 원하는<br />
+                상품을 검색하세요
+              </h2>
+              <p>
+                구매하고 싶은 물품은 검색해서<br />
+                쉽게 찾아보세요
+              </p>
+            </div>
+            <img src="img/Img_home_02.png" />
+          </div>
+        </article>
+        <article class="main-article">
+          <div class="main-article-container">
+            <img src="img/Img_home_03.png" />
+            <div class="main-article__text">
+              <h5>Register</h5>
+              <h2>
+                판매를 원하는<br />
+                상품을 등록하세요
+              </h2>
+              <p>어떤 물건이든 판매하고 싶은 상품을 쉽게 등록하세요</p>
+            </div>
+          </div>
+        </article>
+      </section>
+      <section class="main-space"></section>
+      <section class="main-banner">
+        <banner>
+          <div class="main-banner__content main-banner-bottom__content">
+            <h1>
+              믿을 수 있는<br />
+              판다마켓 중고 거래래
+            </h1>
+          </div>
+          <img src="img/Img_home_bottom.png" />
+        </banner>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-container">
+        <div>&copy;codeit - 2024</div>
+        <div class="footer-middle">
+          <a href="/privacy">privacy Policy</a>
+          <a href="/faq">FAQ</a>
+        </div>
+        <div class="footer-right">
+          <a href="https://www.facebook.com/" target="_blank"
+            ><img src="img/ic_facebook.png"
+          /></a>
+          <a href="https://x.com/" target="_blank"
+            ><img src="img/ic_twitter.png"
+          /></a>
+          <a href="https://www.youtube.com/" target="_blank"
+            ><img src="img/ic_youtube.png"
+          /></a>
+          <a href="https://www.instagram.com/" target="_blank"
+            ><img src="img/ic_instagram.png"
+          /></a>
+        </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="footer-right">
          <li>
            <a href="https://www.facebook.com/" target="_blank">
              <img src="img/ic_facebook.png"/>
            </a>
          </li>
        </ul>
```